reporter/html.js

```diff
@@ -844,7 +860,7 @@ export function escapeText( s ) {
 	} );
 
 	QUnit.testDone( function( details ) {
-		var testTitle, time, testItem, assertList,
+		var testTitle, time, testItem, assertList, status,
```

Comment:
Why do we need `status` again? Seems redundant to details.<status>

Reply:
It was more of a style thing, we mutate message a ton I just thought it was a needed abstraction

Reply:
I can change it back